package.json

```diff
@@ -27,22 +28,26 @@
     "roar-mp",
     "roar-pa"
   ],
-  "author": "Shoaib Mohammed",
+  "author": "Shoaib Mohammed and Philip Ziolkowski",
```

Comment:
Let's remove the author field and adopt a `contributors` field, using the more verbose object format.